.github/workflows/semantic-breaking-change-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Semantic Breaking Change PR
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
```

Comment:
This action run won't change its decision after the label removed. The `checkDescription` function exits if PR has no `breaking changes` label, so even if the `breaking changes` label is not removed from the PR, this action won't report anything. (maybe I just misunderstood something)

Reply:
@auvred 
 This was to avoid performing a description check when the breaking change label is removed. I thought that if the breaking change label was accidentally added and removed, the CI should rerun and change the status to Pass. Am I misunderstanding something?

Reply:
Ahh, I get it! Now it makes perfect sense to me! Thanks for the explanation!